README.md

```diff
@@ -32,18 +32,18 @@ We will have to do this once:
 After all this, start the bot with `npm run start`.
 
 > ### ⚠ Warning 
-> There is literally no warranty if you self-host Countr, and we will not help you set it up either. If you wish to set the bot up yourself, we expect you have well enough knowledge in Node.js. We still recommend using the original bot.
+> There is no warranty if you self-host Countr, and we will not be helping you set it up either. If you wish to host the bot yourself, we expect you have to have the necessary knowledgee of Node.js. We still recommend using the original bot.
```

Comment:
would recommend reverting be helping to help , knowledge has one too many e's.

Reply:
eh it looks fine